Content.Shared/_White/Collision/Blur/BlurOnCollideComponent.cs

```diff
@@ -0,0 +1,8 @@
+namespace Content.Shared._White.Collision.Blur;
+
+[RegisterComponent]
+public sealed partial class BlurOnCollideComponent : Component
+{
+    [DataField]
+    public float BlurTime = 20f;
```

Comment:
Для одного выстрела из арбалета с кд в секунды 3 вроде долговато.
```suggestion
    public float BlurTime = 10f;
```